module/solutions/helper.js

```diff
@@ -2235,7 +2232,7 @@ module.exports = class SolutionsHelper {
 	 * @returns {Object} List of library projects.
 	 */
 
-	static projects(query, pageSize, pageNo, searchQuery, fieldsArray, groupBy = '') {
```

Comment:
@Prajwal17Tunerlabs keep the page size and pageno. if they are passed as empty take default page size and number

Reply:
ok, keeping pageSize and pageNo variables